Model/src/main/java/org/eupathdb/common/model/report/ZippedFilesReporter.java

```diff
@@ -54,7 +54,7 @@ public void report(OutputStream stream) throws WdkModelException {
         ZipEntry file = new ZipEntry(record.getAttributeValue(FILENAME_ATTRIBUTE).getValue());
         out.putNextEntry(file);
         String clientUrl = record.getAttributeValue(FILEPATH_ATTRIBUTE).getValue();
-        String absoluteFileUrl = _answerValue.getWdkModel().getProperties().get(MODEL_PROP_LOCALHOST) + clientUrl.replace("a/app", "/common");
+        String absoluteFileUrl = _answerValue.getWdkModel().getProperties().get(MODEL_PROP_LOCALHOST) + "/" + clientUrl;
```

Comment:
Sure?  Bindu promised "common" without the leading slash, right?

Reply:
I guess I should verify with her. I could also make this more defensive and only add the leading slash if it isn't present.

Reply:
I talked to Bindu, and she added the leading slash in the database.